x-pack/plugins/index_management/public/application/sections/home/index_list/details_page/details_page_overview/details_page_overview.tsx

```diff
@@ -65,99 +73,28 @@ export const DetailsPageOverview: React.FunctionComponent<Props> = ({ indexDetai
     indexName: name,
   };
 
+  const isLarge = useIsWithinBreakpoints(['xl']);
+
   return (
     <>
-      <EuiFlexGroup>
-        {config.enableIndexStats && (
-          <EuiFlexItem data-test-subj="overviewTabIndexStats">
-            <EuiPanel>
-              <EuiFlexGroup>
-                <EuiFlexItem>
-                  <EuiStat
-                    title={status}
-                    titleColor={status === 'open' ? 'success' : 'danger'}
-                    description={i18n.translate(
-                      'xpack.idxMgmt.indexDetails.overviewTab.statusLabel',
-                      {
-                        defaultMessage: 'Status',
-                      }
-                    )}
-                  />
-                </EuiFlexItem>
-                <EuiFlexItem>
-                  <EuiStat
-                    title={documents}
-                    titleColor="primary"
-                    description={i18n.translate(
-                      'xpack.idxMgmt.indexDetails.overviewTab.documentsLabel',
-                      {
-                        defaultMessage: 'Documents',
-                      }
-                    )}
-                  />
-                </EuiFlexItem>
-                <EuiFlexItem>
-                  <EuiStat
-                    title={documentsDeleted}
-                    description={i18n.translate(
-                      'xpack.idxMgmt.indexDetails.overviewTab.documentsDeletedLabel',
-                      {
-                        defaultMessage: 'Documents deleted',
-                      }
-                    )}
-                  />
-                </EuiFlexItem>
-              </EuiFlexGroup>
-            </EuiPanel>
+      <EuiFlexGrid columns={isLarge ? 3 : 2}>
```

Comment:
If we ever run into a scenario in which we only have one column (for example in serverless), is there any chance we could show some other data so that it wouldnt end up being that empty?

Otherwise I think might look better if we just use one column and have the only card to be expanded:

![Screenshot 2023-10-27 at 13 22 52](https://github.com/elastic/kibana/assets/1191206/595f3414-95a8-47d8-aa74-33ccb90f32ed)

Reply:
On serverless there is no other info to display, so it might be either nothing, or only aliases card, or only data stream card. I personally would prefer the card to stay as a 1/3 of the width even when there is only 1 card. But let me get some feedback from the design team on that.

Reply:
Yeah I'll leave it up design on what to do 👍🏼

Reply:
Sent your screenshot to the design team and will follow up in a separate PR if needed